tests/sentry/web/frontend/test_react_page.py

```diff
@@ -402,7 +395,7 @@ def test_customer_domain_org_deletion_in_progress(self):
             assert response.redirect_chain == [
                 ("http://testserver/organizations/new/", 302),
             ]
-            assert "activeorg" not in self.client.session
+            assert "activeorg" in self.client.session
```

Comment:
This flip and the one on 379 are how the system was always working. However, the results were opposite before because `SENTRY_USE_CUSTOMER_DOMAINS` was `False`. Because of that, the logic that sets `activeorg` would not have run before.